certificate-policy.md

```diff
@@ -600,90 +599,117 @@ The CA SHALL disclose all Cross Certificates that identify the CA as the Subject
 ## 4.1 Certificate Application
 
 ### 4.1.1 Who can submit a certificate application
-In accordance with Section 5.5.2, the CA SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. The CA SHALL use this information to identify subsequent suspicious certificate requests.
+In accordance with Section 5.5.2, all CAs SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. All CA SHALL use this information to identify subsequent suspicious certificate requests.
+
+For the Root and Subordinate CAs:
+- An application for a CA certificate shall be submitted by an authorized representative of the applicant CA.
+
+For end entity certificates:
+- A certificate application shall be submitted to the CA by the Subscriber, an authorized organization representative, or an RA on behalf of the Subscriber. 
```

Comment:
The DoD issuing CA will put the specific of the process in the CPS (as should any CA that operates under this policy.  It will state how the "authority" is determined.  But to be allowed to do that in the CPS, there needs to be enabling text in the CP.  If there isn't something that indicates that an  "authorized organization representative" is allowed then the CPS could be judges non-complaint with the CP - and fail the audit.